### PR TITLE
feature: allow disable ssl for minio

### PR DESCRIPTION
**Pull request by mtatheonly**
allows disabling ssl for minio, for a local/docker installation without ssl enabled

### CODE REVIEW

File: docs/content/how-to/uploading-images-to-minio.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Then configure your `medusa-config.js` to include the plugin alongside the requi
         bucket: "test",
         access_key_id: "YOUR-ACCESS-KEY",
         secret_access_key: "YOUR-SECRET-KEY",
+	sslEnabled: true || false, // if you're running locally with no ssl certificate, use false. Defaults to true if not set
```
Comment (olivermrbl) [inline docs/content/how-to/uploading-images-to-minio.md:43]: issue: Should be snake_case like other options

File: packages/medusa-file-minio/README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ Upload files to a MinIO server.
   bucket: [name of your bucket],
   access_key_id: [access-key],
   secret_access_key: [secret-access-key],
+  sslEnabled: [if ssl is enabled or not]
```
Comment (olivermrbl) [inline packages/medusa-file-minio/README.md:12]: issue: Should be snake_case like other options

File: packages/medusa-file-minio/src/services/minio.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ class MinioService extends FileService {
     this.endpoint_ = options.endpoint
     this.s3ForcePathStyle_ = true
     this.signatureVersion_ = "v4"
+    this.sslEnabled_ = ('sslEnabled' in options && options.sslEnabled !== undefined) ? options.sslEnabled : true
```
Comment (olivermrbl) [inline packages/medusa-file-minio/src/services/minio.js:15]: suggestion: I believe we could simplify this
```
this.sslEnabled = !!options?.sslEnabled
```

**Review by olivermrbl — COMMENTED** [review]
question: Do you think we should enable SSL by default?

**Comment by mtatheonly** [comment]
@olivermrbl, yes, I think it should be default

File: packages/medusa-file-minio/src/services/minio.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ class MinioService extends FileService {
     this.endpoint_ = options.endpoint
     this.s3ForcePathStyle_ = true
     this.signatureVersion_ = "v4"
+    this.sslEnabled_ = !!options?.ssl_enabled
```
Comment (olivermrbl) [inline packages/medusa-file-minio/src/services/minio.js:15]: issue: Does not default to true even though documentation says so
```suggestion
    this.sslEnabled_ = options?.ssl_enabled ? options.ssl_enabled : false
```

**Review by olivermrbl — CHANGES_REQUESTED** [review]

**Comment by riqwan** [comment]
Hey, thanks for the PR! Since v2 brought a lot of architectural and API changes on the backend, we will be closing this ticket.

**Pull request closed without merging**